app/views/EabServicesProvidedView.scala.html

```diff
@@ -25,7 +25,7 @@
 @(form: Form[Seq[EabServicesProvided]], mode: Mode)(implicit request: Request[_], messages: Messages)
 
 @main_template(
-    title = s"${errorPrefix(form)} ${messages("eabServicesProvided.title")}"
+    title = s"${errorPrefix(form)} ${messages("eabServicesProvided.title")} - Estate agency business - Manage your anti-money laundering supervision - GOV.UK".trim
```

Comment:
Could you put "- Estate agency business - Manage your anti-money laundering supervision - GOV.UK" in messages file for me, please? And just use the key? - the same in views below.

Reply:
Done